base/src/main/scala/co/uproot/abandon/BuildInfo.scala

```diff
@@ -0,0 +1,20 @@
+package co.uproot.abandon
+
+import scala.Predef._
+
+/** This object was generated by sbt-buildinfo. */
```

Comment:
@alexanderzafirov This generated file should not be checked in to the repository.